howto/hybrid-clusters/README.md

```diff
@@ -0,0 +1,28 @@
+# Deploying to Hybrid Linux/Winodws K8s Clusters
```

Comment:
Note case and Kubernetes full name:
Deploying to hybrid Linux/Winodws Kubernetes clusters

Reply:
Will fix.